CHANGELOG.md

```diff
@@ -30,14 +30,15 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ### Changed
 
 - Cursors are now inverted when their fixed color is similar to the cell's background
+- Use working directory of the terminal foreground process instead of shell for SpawnNewInstance action
```

Comment:
```suggestion
- Use the working directory of the terminal foreground process, instead of the shell's working
    directory, for the `SpawnNewInstance` action
```